### PR TITLE
Exercise: add database schema diagram

### PR DESCRIPTION
**Pull request by sarazaToussaint**
## In this pull request we:

- Interpret and implement the SQL needed to build the database with tables based on diagram.

### CODE REVIEW

File: schema_based_on_diagram.sql [hunk 1]
```diff
@@ -0,0 +1,49 @@
+/* Database schema to keep the structure of entire database. */
```
Comment (omar-labana) [inline schema_based_on_diagram.sql:1]: - [x] all FOREIGN KEYS should have indexes created. please add them.

**Review by omar-labana — CHANGES_REQUESTED** [review]
Hi team, 

Excellent work so far! 👌
You have implemented almost all the required features. ✔️ 
There are some cases that you need to adjust but you are almost done! 😀

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by TedLivist — APPROVED** [review]
## STATUS: APPROVED ✔️ ✔️

Hi @sarazaToussaint and @bamsi,

✔️ Complete database structure based on diagram
✔️ Implementation of indexes for foreign keys

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: schema_based_on_diagram.sql (hunk 1 above)
Comment (sarazaToussaint) [reply schema_based_on_diagram.sql:1]: Hi @omar-labana, thank you for the code review.

**Comment by sarazaToussaint** [comment]
Hi @TedLivist, thank you for the code review.

**Pull request merged**